src/compute/src/render/join/linear_join.rs

```diff
@@ -523,43 +531,45 @@ where
 {
     match (prev_keyed, next_input) {
         (
-            SpecializedArrangementImport::Bytes9Row(prev_key_types, prev_keyed),
-            SpecializedArrangementImport::Bytes9Row(next_key_types, next_input),
-        ) => {
-            assert!(prev_key_types
-                .iter()
-                .zip(next_key_types.iter())
-                .all(|(c1, c2)| c1.scalar_type == c2.scalar_type));
-            differential_join_inner(
-                join_spec,
-                prev_keyed,
-                next_input,
-                Some(prev_key_types),
-                None,
-                None,
-                closure,
-            )
-        }
+            SpecializedArrangementImport::RowUnit(prev_keyed),
+            SpecializedArrangementImport::RowUnit(next_input),
+        ) => differential_join_inner(
+            join_spec,
+            prev_keyed,
+            next_input,
+            None,
+            Some(vec![]),
+            Some(vec![]),
+            closure,
+        ),
```

Comment:
Uncovered

Reply:
Same as above; please reopen this conversation if not true.